File: Java/extra/Lista 01/src/ex08.java
//Lê o saldo de uma aplicação e imprima o novo saldo, considerado o reajuste de 1%.

import java.util.Scanner;

public class ex08 {
    static double reajuste(double a) {
        return a * 1.1;
    }

    public static void main(String[] args) {
        double num;
        try (Scanner leitor = new Scanner(System.in)) {
            System.out.println("Digite o valor que sofrerá o reajuste: ");
            num = leitor.nextDouble();
        }

        System.out.format("O valor %.2f com reajuste de 1%% ficará %.2f", num, reajuste(num));
    }
}

File: C/Listas/Lista 01/ex08.c
//Lê o saldo de uma aplicação e imprima o novo saldo, considerado o reajuste de 1%.
#include <stdio.h>

int main(void){
    float saldo;
    printf("Digite o valor do saldo: ");
    scanf("%f", &saldo);
    
    printf("\nSaldo inserido: R$%.1f", saldo);
    printf("\nO novo saldo considerando um reajuste de 1%% passará a ser R$%.1f", saldo+(saldo*0.01));
}